### PR TITLE
New version: FastCopy ver 4.0.1

### PR DESCRIPTION
**Pull request by matcha-shake**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41853)

### CODE REVIEW

File: manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: FastCopy.FastCopy
+PackageVersion: 4.0.1
+PackageName: FastCopy
+Publisher: H.Shirouzu
```
Comment (Trenly) [inline manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml:7]: The Publisher needs to match what is in control panel
```suggestion
Publisher: H.Shirouzu & FastCopy Lab, LLC.
```

File: manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: FastCopy.FastCopy
+PackageVersion: 4.0.1
+PackageName: FastCopy
+Publisher: H.Shirouzu
+License: GPL 3.0
```
Comment (Trenly) [inline manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml:8]:
```suggestion
License: GPL-3.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml (hunk 1 above)
Comment (matcha-shake) [reply manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml]: Thank you for the suggestion.
I update the file based on the software license page.
https://fastcopy.jp/help/fastcopy_eng.htm#license

File: manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml (hunk 2 above)
Comment (matcha-shake) [reply manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml]: Thank you for the suggestion.
I update the file based on the software license page.
https://fastcopy.jp/help/fastcopy_eng.htm#license

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Package was merged: https://github.com/microsoft/winget-pkgs/blob/master/manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml

**Comment by matcha-shake** [comment]
Closed. Because the commit created by the FastCopy's auther has been merged.

**Pull request closed without merging**